From 44e3ce1d066e2bcc574081ed828368974e0262a2 Mon Sep 17 00:00:00 2001
From: Tobias Christiansen <tobyase@serenityos.org>
Date: Fri, 11 Mar 2022 19:01:35 +0100
Subject: [PATCH 2/3] Stub out the performance report macro

The Macro for performance reporting was throwing compile errors, so we just stub it out.
---
 src/Performance.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Performance.cc b/src/Performance.cc
index e77b6da..1d6ff6e 100644
--- a/src/Performance.cc
+++ b/src/Performance.cc
@@ -224,7 +224,7 @@ const uint64_t subsq_avg_AB = Statistics_record::average(sum_subsq_cycles_AB,
    //
 #define perfo_1(id, ab, _name, _thr)
 #define perfo_2(id, ab, _name, _thr)
-#define perfo_3(id, ab, _name, _thr) fs_ ## id ## ab.print(out);
+#define perfo_3(id, ab, _name, _thr)
 #define perfo_4(id, ab, name, thr) perfo_3(id, ab, name, thr)
 
 #include "Performance.def"
-- 
2.36.1

